modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite3.java

```diff
@@ -138,10 +138,6 @@
         //GridTestUtils.addTestIfNeeded(suite,GridCacheReplicatedTxMultiNodeBasicTest.class, ignoredTests);
         //GridTestUtils.addTestIfNeeded(suite,GridCacheReplicatedTxReadTest.class, ignoredTests);
 
-        // TODO GG-11141.
-//        GridTestUtils.addTestIfNeeded(suite,GridCacheDeploymentSelfTest.class, ignoredTests);
```

Comment:
Class is removed according to issue reporter suggestion "these tests were testing peer class loading for cache entry (key/value). This case is not to be supported in the nearest future".